spec/jobs/whitehall_document_import_job_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe WhitehallDocumentImportJob do
+  include ActiveJob::TestHelper
+
+  let(:whitehall_migration) {
+    build(:whitehall_migration)
+  }
+
+  let(:whitehall_migration_document_import) {
+    build(:whitehall_migration_document_import, whitehall_migration_id: whitehall_migration["id"])
+  }
+
+  before do
+    allow(WhitehallImporter).to receive(:import_and_sync).and_return("return_value")
+    allow(WhitehallImporter).to receive(:mark_migration_completed).and_return("return_value")
```

Comment:
Do we use/need these "return_value" declarations?

Reply:
Removed these since they aren't needed.